module/Company/src/Company/Model/CompanyI18n.php

```diff
@@ -0,0 +1,223 @@
+        $this->company = $company;
+    }
+    
+    /**
+     * Get the company's slogan.
+     *
+     * @return string
+     */
+    public function getSlogan()
+    {
+        return $this->slogan;
+    }
+
+    /**
+     * Set the company's slogan.
+     *
+     * @param string $slogan
+     */
+    public function setSlogan($slogan)
+    {
+        $this->slogan = $slogan;
+    }
+
+    /**
+     * Get the company's logo.
+     *
+     * @return string
+     */
+    public function getLogo()
+    {
+        return $this->logo;
+    }
+
+    /**
+     * Set the company's logo.
+     *
+     * @param string $logo
+     */
+    public function setLogo($logo)
+    {
+        $this->logo = $logo;
+    }
+
+    /**
+     * Get the company's description.
+     *
+     * @return string
+     */
+    public function getDescription()
+    {
+        return $this->description;
+    }
+
+    /**
+     * Set the company's description.
+     *
+     * @param string $description
+     */
+    public function setDescription($description)
+    {
+        $this->description = $description;
+    }
+    
+    /**
+     * Get the company's website.
+     *
+     * @return string
+     */
+    public function getWebsite()
+    {
+        return $this->website;
+    }
+
+    /**
+     * Set the company's website.
+     *
+     * @param string $website
+     */
+    public function setWebsite($website)
+    {
+        $this->website = $website;
+    }
+
+    /**
+     * Get the company's language.
+     *
+     * @return string
+     */
+    public function getLanguage()
+    {
+        return $this->language;
+    }
+
+    /**
+     * Set the company's language.
+     *
+     * @param string $language
+     */
+    public function setLanguage($language)
+    {
+        $this->language = $language;
+    }
+    
+    // For zend2 forms
+    public function getArrayCopy()
+    {
+        return get_object_vars($this);
+    }
+    
+    public function exchangeArray($data) {
+        $this->name=(isset($data['name'])) ? $data['name'] : $this->getName();
+        $this->slugName=(isset($data['slugName'])) ? $data['slugName'] : $this->getSlugName();
+        $this->address=(isset($data['address'])) ? $data['address'] : $this->getAddress();
+        $this->website=(isset($data['website'])) ? $data['website'] : $this->getWebsite();
+        $this->slogan=(isset($data['slogan'])) ? $data['slogan'] : $this->getSlogan();
+        $this->email=(isset($data['email'])) ? $data['email'] : $this->getEmail();
+        $this->logo=(isset($data['logo'])) ? $data['logo'] : $this->getLogo();
+        $this->phone=(isset($data['phone'])) ? $data['phone'] : $this->getPhone();
+        $this->description=(isset($data['description'])) ? $data['description'] : $this->getDescription();
+        $this->jobs=(isset($data['jobs'])) ? $data['jobs'] : $this->getJobs();
+        $this->packet=(isset($data['packet'])) ? $data['packet'] : $this->getPacket();
```

Comment:
Keep coding standards in mind! (PSR-2)

Also, this looks like you can do this in a much more readable way.